src/views/List.jsx

```diff
@@ -18,29 +24,53 @@ export function List({ data }) {
 		setSearchTerm('');
 	};
 
+	const copyToken = () => {
+		navigator.clipboard.writeText(listToken);
+		setCopied(true);
+	};
```

Comment:
Great idea!

Reply:
It would be a good idea to handle copy to clipboard failures here, in case the user has denied clipboard permissions. See https://developer.mozilla.org/en-US/docs/Web/API/Clipboard